src/buffer/out/textBuffer.cpp

```diff
@@ -1176,19 +1176,26 @@ void TextBuffer::Reset() noexcept
     _initialAttributes = _currentAttributes;
 }
 
+// Arguments:
+// - start: The y-position of the viewport. We'll clear up until here.
+// - height: the number of rows to keep in the buffer.
 void TextBuffer::ClearScrollback(const til::CoordType start, const til::CoordType height)

```

Comment:
WHAT ON EARTH
I move that we rename both of these parameters, before they do more psychic damage to the team

Reply:
* `clearUpToThisRow`, `numberOfRowsToKeep`? 
* `currentViewportTop`, `viewportHeight`?
* ?

Reply:
(Also sorry for sending you chasing untamed ornithoids about `start`)